src/adapters/lokijs/worker/encodeQuery/index.js

```diff
@@ -86,6 +87,19 @@ const noNullComparisons: OperatorFunction => OperatorFunction = operator => valu
   $and: [operator(value), weakNotEqual(null)],
 })
 
+const like: OperatorFunction = value => {
+  if (typeof value === 'string') {
+    const regexp = `^${value}$`.replace(/%/g, '.*').replace(/_/g, '.')
+    return {
+      $regex: new RegExp(regexp, 'i'),
+    }
+  }
+
+  return {
+    $regex: new RegExp('(?!.*)', 'i'),
```

Comment:
Good idea :) I think `(?!)` should be enough and as this regex does not have dynamic value it is better to use literal. `/(?!)/i`